src/settings/JobProfiles/JobProfileDetails/JobProfileDetails.js

```diff
@@ -10,157 +7,89 @@ import {
-                        <ViewMetaData
-                          metadata={jobProfile.metadata}
-                          stripes={stripes}
-                        />
-                        <Row>
-                          <Col xs>
-                            <KeyValue
-                              data-test-job-profile-name
-                              label={<FormattedMessage id="ui-data-export.name" />}
-                              value={jobProfile.name}
-                            />
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col xs>
-                            <KeyValue
-                              data-test-job-profile-mapping-profile
-                              label={<FormattedMessage id="ui-data-export.mappingProfile" />}
-                              value={mappingProfile.name}
-                            />
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col xs>
-                            <KeyValue
-                              data-test-job-profile-protocol
-                              label={<FormattedMessage id="ui-data-export.protocol" />}
-                              value={<NoValue />}
-                            />
-                          </Col>
-                        </Row>
-                        <Row>
-                          <Col xs>
-                            <KeyValue
-                              data-test-job-profile-description
-                              label={<FormattedMessage id="ui-data-export.description" />}
-                              value={jobProfile.description || <NoValue />}
-                            />
-                          </Col>
-                        </Row>
-                      </Accordion>
-                    </AccordionSet>
-                  </AccordionStatus>
-                  <ConfirmationModal
-                    id="delete-job-profile-confirmation-modal"
-                    open={isConfirmationModalOpen}
-                    heading={<FormattedMessage id="ui-data-export.jobProfiles.delete.confirmationModal.title" />}
-                    message={(
-                      <SafeHTMLMessage
-                        id="ui-data-export.jobProfiles.delete.confirmationModal.message"
-                        values={{ name: jobProfile.name }}
-                      />
-                    )}
-                    confirmLabel={<FormattedMessage id="ui-data-export.delete" />}
-                    cancelLabel={<FormattedMessage id="ui-data-export.cancel" />}
-                    onCancel={() => setConfirmationModalState(false)}
-                    onConfirm={() => handleDelete(jobProfile)}
+    <ProfileDetails
+      profile={jobProfile}
+      type="job"
+      {...props}
+    >
+      { !isLoading && (
```

Comment:
Just for consistency. It is a pity that eslint allows this.
```suggestion
      {!isLoading && (
```